### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by fffw**
Add instruction to launch a new proxy

### CODE REVIEW

**Comment by fffw** [comment]
@aranhoide I just added my steps to create a new proxy. Please revise if there's anything missing or incorrect.

File: README.md [hunk 1]
```diff
@@ -245,6 +245,32 @@ Your cloudmaster should be ready now.  If it's not a production one (XXX: add in
...
+First, add an request to the queue.
+```python
+from redis_util import redis_shell as r
+r.delete('donyc3myname:srvreqq') # run multiple times until it returns 0
+r.lpush('donyc3myname:srvreqq', -1) # end mark
+r.lpush('donyc3myname:srvreqq', 1) # can be any positive number not already there
+r.lrange('donyc3myname:srvreqq', 0, -1) # to verify, the result should be [1, -1]
+```
+
+Then run the refill service.
```
Comment (aranhoide) [inline README.md]: As of `master`, the `refill_cm_srvq` service should be enabled in all cloudmasters, test or production, so this step shoudn't be necessary.

File: README.md [hunk 2]
```diff
@@ -245,6 +245,32 @@ Your cloudmaster should be ready now.  If it's not a production one (XXX: add in
 
 [2] You can't use `bin/hscloudmaster.bash` here because it hasn't been adapted to work as root, which is only needed during this bootstrap procedure.
 
+##### Launching a proxy
+
+First, add an request to the queue.
+```python
+from redis_util import redis_shell as r
+r.delete('donyc3myname:srvreqq') # run multiple times until it returns 0
```
Comment (aranhoide) [inline README.md]: `delete` deletes the whole queue, so you should never need do it more than once.  If it ever gets refilled, this is probably the local refill_cm_srvq crashing and respawning, which creates the `-1` sentinel value again.

Comment (aranhoide) [inline README.md]: These instructions have the advantage of being robust to whatever is the current state of the queue, but the only step that should be needed in normal usage is adding the new request.

**Comment by aranhoide** [comment]
@fffw : this procedure is probably more work than necessary in most cases, but has the advantage that it should work in all cases, except for one thing: if your cloudmaster is up-to-date, it should already be running a `refill_cm_srvq` service, which will compete with your shell-spawned one to grab the job.  So spawning a shell version of the service should only ever be needed for debugging the process of launching proxies.

**Comment by fffw** [comment]
Thanks for the details review @aranhoide . Updated based on your comments

**Comment by aranhoide** [comment]
LGTM, thanks!

**Pull request merged**